app/connectors/CustomsDeclareExportsMovementsConnector.scala

```diff
@@ -65,22 +64,22 @@ class CustomsDeclareExportsMovementsConnector @Inject()(appConfig: AppConfig, ht
   )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] =
     postRequest(DucrDisassociation, requestXml)
 
-  def sendShutMucrRequest(requestXml: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] =
-    postRequest(ShutMucr, requestXml)
-
   private def postRequest(
     actionType: ActionType,
-    requestXml: String
+    payload: String
   )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] =
     httpClient
-      .POSTString[HttpResponse](movementSubmissionUrl(actionType), requestXml, CommonMovementsHeaders)
+      .POSTString[HttpResponse](movementSubmissionUrl(actionType), payload, CommonMovementsHeaders)
       .andThen {
         case Success(response) =>
           logger.debug(s"CUSTOMS_DECLARE_EXPORTS_MOVEMENTS response on ${actionType.value}. $response")
         case Failure(exception) =>
           logger.warn(s"CUSTOMS_DECLARE_EXPORTS_MOVEMENTS failure on ${actionType.value}. $exception ")
       }
 
+  def sendShutMucrRequest(requestXml: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] =
+    postRequest(ShutMucr, requestXml)
```

Comment:
The point of this was to get rid of the xml from the front end :)
Can we change this to submit the MovementRequest

Reply:
This is the work @PatrykRudnicki is working on and I did not want to change it, to avoid code conflicts. This has been already removed by his work